src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/Section/FieldInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section;
+
+/**
+ * @author Edouard Lescot <edouard.lescot@gmail.com>
+ * @author Oskar Stark <oskarstark@googlemail.com>
+ */
+interface FieldInterface
```

Comment:
same here, never typehinted/used ?

Reply:
No but used as a marker interface, I would like to keep it

Reply:
I'm for removing it, it's not used at all. We can consider re-adding it later if a legit use case comes out.